ci/pipeline.yml

```diff
@@ -1,7 +1,7 @@
 #@ load("@ytt:data", "data")
 
 #@ def pipeline_image():
-#@   return data.values.docker_registry + "/galoy-app-pipeline"
+#@   return data.values.docker_registry + "/nodejs-concourse"
```

Comment:
What's the context for this change?

Reply:
`/nodejs-concourse` is our new base Docker image for doing tests